x/ccv/provider/ibc_module.go

```diff
@@ -175,28 +175,49 @@ func (am AppModule) OnRecvPacket(
 	_ sdk.AccAddress,
 ) ibcexported.Acknowledgement {
 	var (
-		ack            ibcexported.Acknowledgement
-		consumerPacket ccv.ConsumerPacketData
+		ack              ibcexported.Acknowledgement
+		consumerPacket   ccv.ConsumerPacketData
```

Comment:
I think we can avoid the flag and multiple vars here by converting to one version or another early on

Reply:
Done!